db/migrate/20230830041832_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: { to_table: :users }
```

Comment:
- [ ] It would be nice if you update the references to the user table because the user table has author_id as an alias of user_id so it would be nice if you make your association to the user table with an alias author instead of user 👍 to follow the project requirements 😊

As you can see below ⬇️

```ruby
t.references :author, null: false, foreign_key: { to_table: :users }
```